src/Equinor.ProCoSys.IPO.WebApi/Synchronization/BusReceiverService.cs

```diff
@@ -154,15 +154,23 @@ private void ProcessCommPkgEvent(string messageJson)
             }
 
             if (string.IsNullOrWhiteSpace(commPkgEvent.Plant)  ||
-                commPkgEvent.ProCoSysGuid == Guid.Empty ||
-                commPkgEvent.ProjectGuid == Guid.Empty)
+                commPkgEvent.ProCoSysGuid == Guid.Empty)
             {
-                throw new Exception($"Key attributes Plant, ProCoSysGuid and/ or ProjectGuid is not provided in CommPkgEvent message: {messageJson}");
+                throw new Exception($"Key attributes Plant and/ or ProCoSysGuid is not provided in CommPkgEvent message: {messageJson}");
+            }
+
+            if (!_invitationRepository.IsExistingCommPkg(commPkgEvent.ProCoSysGuid))
+            {
+                throw new Exception($"Given ProCoSysGuid does not refer to an existing object. CommPkgEvent message: {messageJson}");

```

Comment:
Maybe clarify that there is no CommPkg with the given ProCoSysGuid instead of simply an object.